contracts/truefi2/FixedTermLoanAgency.sol

```diff
@@ -540,19 +508,6 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
         revert("FixedTermLoanAgency: This loan has not been funded by the agency");
     }
 
-    /// @dev Helper used in tests
-    function _distribute(
-        address recipient,
-        uint256 numerator,
-        uint256 denominator,
-        address pool
-    ) internal {
-        ILoanToken2[] storage _loans = poolLoans[ITrueFiPool2(pool)];
-        for (uint256 index = 0; index < _loans.length; index++) {
-            _transferLoan(_loans[index], recipient, numerator, denominator);
-        }
-    }
-
     // @dev Transfer (numerator/denominator)*balance of loan to the recipient
     function _transferLoan(
```

Comment:
We probably could have inlined this function as well. It's only called in one place, with numerator == denominator == 1. So `loan.safeTransfer(recipient, loan.balanceOf(address(this)))` would have been good enough instead of an 8 line function.